llvm/test/CodeGen/WebAssembly/unreachable.ll

```diff
@@ -1,33 +1,133 @@
-; RUN: llc < %s -asm-verbose=false -verify-machineinstrs | FileCheck %s
-; RUN: llc < %s -asm-verbose=false -fast-isel -fast-isel-abort=1 -verify-machineinstrs | FileCheck %s
-
-; Test that LLVM unreachable instruction and trap intrinsic are lowered to
-; wasm unreachable
+; RUN: llc < %s -verify-machineinstrs | FileCheck %s
```

Comment:
I think this file needs some more comments on what this file tries to test. It is hard to figure out what this file is trying to test. It looks all tests just produce Wasm `unreachable` for LLVM `unreachable` and nothing else, and it is hard to figure out why we need all these different tests and all these combinations of this options for, unless they want to go through all the discussions in this PR. Even after all the discussions even I'm not sure why we need all these added tests tbh.